content/en/tutorials/simulating-outages/index.md

```diff
@@ -0,0 +1,224 @@
+In this example, we will use the Chaos Plugin to create controlled outages in a DynamoDB database.
+The aim is to test the software's behavior and error handling capabilities.
+
+For this particular example, we'll be using a [sample application repository](https://github.com/localstack-samples/samples-chaos-engineering/tree/master/chaos-plugin).
+Clone the repository, and follow the instructions below to get started.
+
+### Prerequisites
+
+The general prerequisites for this guide are:
+
+- LocalStack Pro with [LocalStack Auth Token]({{<ref "getting-started/auth-token">}})
+- [AWS CLI]({{<ref "user-guide/integrations/aws-cli">}}) with the [`awslocal` wrapper]({{<ref "user-guide/integrations/aws-cli#localstack-aws-cli-awslocal">}})
+- [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/)
+
+Start LocalStack by using the `docker-compose.yml` file from the repository.
+Ensure to set your Auth Token as an environment variable during this process.
+The cloud resources will be automatically created upon the LocalStack start.
+
+{{< command >}}
+$ LOCALSTACK_AUTH_TOKEN=<YOUR_LOCALSTACK_AUTH_TOKEN>
+$ docker compose up
+{{< /command >}}
+
+### Architecture
+
+The following diagram shows the architecture that this application builds and deploys:
+
+{{< figure src="arch-1.png" width="800">}}
+
+### Preflight checks
+
+Before starting any outages, it's important to verify that our application is functioning correctly.
+Start by creating an entity and saving it.
+To do this, use `cURL` to call the API Gateway endpoint for the POST method:
+
+{{< command >}}
+$ curl --location 'http://12345.execute-api.localhost.localstack.cloud:4566/dev/productApi' \
+--header 'Content-Type: application/json' \
+--data '{
+    "id": "prod-2004",
+    "name": "Ultimate Gadget",
+    "price": "49.99",
+    "description": "The Ultimate Gadget is the perfect tool for tech enthusiasts looking for the next level in gadgetry. Compact, powerful, and loaded with features."
+}'
+<disable-copy>
+Product added/updated successfully.
+</disable-copy>
+{{< /command >}}
+
+### Simulating the outage
+
+Next, we will configure the Chaos Plugin to target all APIs of the DynamoDB resource.
+The Chaos Plugin is powerful enough to refine outages to particular operations like `PutItem` or `GetItem`, but the objective here is to simulate a failure of entire database service.
+The following configuration will cause all API calls to fail with a 100% failure rate, each resulting in an HTTP 500 status code and a `DatacentreNotFound` error.
+
+{{<command>}}
+curl --location --request PATCH 'http://localhost.localstack.cloud:4566/_localstack/chaos/faults' \
+--header 'Content-Type: application/json' \
+--data '
+[
+    {
+        "service": "dynamodb",
+        "probability": 1.0,
+        "error": {
+            "statusCode": 500,
+            "code": "DatacentreNotFound"
```

Comment:
this is British spelling?

Reply:
Yes, it's just an example, but I changed it anyway in 60db0d866